packages/dynamite/dynamite/lib/src/openapi_builder.dart

```diff
@@ -970,6 +966,11 @@ TypeResult resolveObject(
                   )
                   ..redirect = refer('_\$${state.prefix}$identifier'),
               ),
+              Constructor(
+                (final b) => b
+                  ..name = '_'
+                  ..constant = true,
+              ),
```

Comment:
Is there a reason this one moved down?

Reply:
Yeah because the formatter would move it down and then break the coverage ignores :roll_eyes: